src/client/public/locale/de/common.json

```
@@ -440,5 +440,7 @@
   "field_measurement_sample_type": "Beprobungstyp",
   "fieldMeasurementRequiredFieldsAlert": "'Beprobungstyp', 'Parameter', 'Wert', 'Beginn' und 'Zuverlässigkeit der Angaben' sind obligatorische Felder, der Eintrag kann nicht gespeichert werden, wenn sie nicht ausgefüllt sind.",
   "msgFieldMeasurementsEmpty": "Keine Feldmessungen vorhanden",
-  "saveHydrotestResultAlert": "Bitte speichern Sie das aktuelle Resultat des Hydrotests, bevor Sie den Hydrotest speichern."
+  "saveHydrotestResultAlert": "Bitte speichern Sie das aktuelle Resultat des Hydrotests, bevor Sie den Hydrotest speichern.",
+  "msgChronostratigraphyEmpty": "Kein Chronostratigraphieprofil verfügbar",
+  "msgChronostratigraphyEmptyEditing": "Sie sollten zuerst eine Stratigraphie erstellen, bevor Sie ein Chronostratigraphieprofil erstellen können"
```

PP: _sollten_ durch _müssen_ ersetzen
```suggestion
  "msgChronostratigraphyEmptyEditing": "Sie müssen zuerst eine Stratigraphie erstellen, bevor Sie ein Chronostratigraphieprofil erstellen können"
```